app/scripts/OverlayTrack.js

```diff
@@ -11,7 +11,130 @@ class OverlayTrack extends PixiTrack {
     this.drawnRects = {};
   }
 
-  drawHorizontalOverlay(graphics, position, extent, minWidth = 0) {
+  drawWithPositionedBorder(
```

Comment:
This doesn't have to be a member function of the class, does it? It has no references to `this`.

Reply:
Good catch. Moved it outside as a context-free function.